example/webdriverio-test/.eslintrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  rules: {
+    'eslint-comments/no-unlimited-disable': 'off',
```

Comment:
Not sure this is necessary. If we want to applay this rule maybe lets use it everywhere?

Reply:
some of webdriverio files use this rule internally. I didn't want to dig too much to understand if this rule can be enabled here, so I just disabled it for now.
Other files in this repo shoudl follow this rule. So I just disabled it locally for these tests.